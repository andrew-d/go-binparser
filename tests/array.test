[
    {"name": "ArrayTest",
     "fields": [
        {"name": "arr", "offset": 0, "type": "uint16[3]"}
    ]}
]
