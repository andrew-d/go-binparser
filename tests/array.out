Arr,[%!s(uint16=13330) %!s(uint16=30806) %!s(uint16=43920)]
